01-factoring-overview.ipynb

```diff
@@ -411,14 +411,9 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# To see helper functions, select Jupyter File Explorer View from the Online Learning page\n",
-    "from helpers.embedding import embeddings\n",
-    "from dwave.system import FixedEmbeddingComposite\n",
-    "\n",
-    "sampler_fixed_embedding = FixedEmbeddingComposite(sampler, embeddings[sampler.solver.name])\n",
+    "from dwave.system import EmbeddingComposite\n",
     "\n",
-    "print(\"Mapping from BQM variables to qubits for the {} system:\\n\\n\\t{}\".format(\n",
-    "      sampler.solver.name, sampler_fixed_embedding.embedding))"
+    "sampler_embedded = EmbeddingComposite(sampler)"
```

Comment:
Maybe `embedding_sampler`, the sampler has not been "embedded".

Reply:
Done